omod/src/main/webapp/resources/scripts/mergepatients/patientSearchAndSelectWidget.js

```diff
@@ -0,0 +1,524 @@
+function PatientSearchWidget(configuration){
```

Comment:
Did all this stuff come from the 'emr' module? There's more custom widget stuff here than I expected.